src/services/diagram-generator.ts

```diff
@@ -12,9 +12,16 @@ const WOLLOK_BASE_MODULES = 'wollok.'
 const SELF = 'self'
 const REPL = 'REPL'
 
+function getImportedDefinitionsFromConsole(interpreter: Interpreter): string[] {
+  const imports = interpreter.evaluation.environment.getNodeByFQN<Package>(REPL).imports
+  return imports.flatMap(imp => imp.children.map(child => (child as unknown as Reference<Entity>).name))
+}
```

Comment:
Esto lo hiciste por nombre, yo pensaba en hacerlo por referencia y después checkear los parent de los modules:

```suggestion
function getImportedDefinitionsFromConsole(interpreter: Interpreter): Package[] {
  return interpreter.evaluation.environment.getNodeByFQN<Package>("REPL").imports.map(imp => imp.entity.target)
}
```

----

Probandolo en la consola CREO que **descubrí un bug** al calcular los imports del archivo a cargar: se están moviendo "así nomás" a un contexto distinto, entonces las referencias relativas se rompen, como al tener 2 archivos en una subcarpeta.

En tu ejemplo, tenés accesso a `fer` y a `ivo` desde la consola cuando cargás `example.wlk`?

Reply:
hola @PalumboN , sí, mi idea originalmente era usar el id, pero como vos decís, algunas referencias relativas se me rompían. Ahora no tengo levantado todo el entorno pero sí veo a `fer` e `ivo` como en el diagrama que muestro en la descripción del PR.

Yo igualmente trataría de no usar los nombres de los archivos, sino los ids internos que generamos de esos objetos.

Reply:
Sí, están en el diagrama, pero no en la consola jajaja. O sea, siguen desfasados, lo que significa que este approach de navegar los imports es bastante frágil. Yo me sigo inclinando por hacerlo desde el `scope`.

Reply:
Pero yo probé y sí estaban en la consola `ivo` y `fer`...

Reply:
Teniendo ambos archivos en una subcarpeta `src`? A mí me está fallando...

Reply:
@PalumboN Ahí hice el cambio que dijiste y está funcionando perfecto, ya no necesitamos hacer el filtro de `true`, `false`, y esas otras cosas de `isLanguageLocal`. Mirá cómo anda:

![dynamicDiagramToqueRasta](https://github.com/uqbar-project/wollok-ts-cli/assets/4549002/466d6c5e-f8a8-4a46-95c2-6290020c2abd)

Reply:
Migré todo a la carpeta src y más allá de que tuve que cambiar el import (esperable), me anduvo joya:

![image](https://github.com/uqbar-project/wollok-ts-cli/assets/4549002/ac15bd2d-e1b8-4c88-b43d-c770c8cb4a1d)

Reply:
Ahh pero para mí no hace falta el cambio del import! 

De hecho de la otra forma anda bien, solamente no anda en la consola (si se referencia dentro del archivo se linkea bien).

Reply:
acá no te seguí, vos querés que cambie algo del código? Porque para mí así está perfecto, solo estoy viendo que hay 2 tests que se rompen, intuyo que porque no es exactamente igual lo que nosotros hacemos en el test vs. lo que ocurre en el REPL de verdad...

Reply:
uh, no, efectivamente ahora estamos filtrando todas las variables, incluso las del REPL